Blockzilla/SettingsViewController.swift

```diff
@@ -182,16 +169,35 @@ class SettingsViewController: UIViewController, UITableViewDataSource, UITableVi
     private lazy var sections = {
         Section.getSections(deviceHasBiometrics: biometryType.hasBiometry)
     }()
-    
-    private var toggles = [
-        BlockerToggle(label: UIConstants.strings.toggleSafari, setting: SettingsToggle.safari),
-        BlockerToggle(label: UIConstants.strings.labelBlockAds, setting: SettingsToggle.blockAds, subtitle: UIConstants.strings.labelBlockAdsDescription),
-        BlockerToggle(label: UIConstants.strings.labelBlockAnalytics, setting: SettingsToggle.blockAnalytics, subtitle: UIConstants.strings.labelBlockAnalyticsDescription),
-        BlockerToggle(label: UIConstants.strings.labelBlockSocial, setting: SettingsToggle.blockSocial, subtitle: UIConstants.strings.labelBlockSocialDescription),
-        BlockerToggle(label: UIConstants.strings.labelBlockOther, setting: SettingsToggle.blockOther, subtitle: UIConstants.strings.labelBlockOtherDescription),
-        BlockerToggle(label: UIConstants.strings.labelBlockFonts, setting: SettingsToggle.blockFonts),
-        BlockerToggle(label: UIConstants.strings.labelSendAnonymousUsageData, setting: SettingsToggle.sendAnonymousUsageData),
-    ]
+
+    private var toggles = [Int : BlockerToggle]()
+
+    private var initialToggles : [Int : BlockerToggle]  {
```

Comment:
Are you able to clarify why we are changing the entire structure of this variable? At first blush it seems more convoluted and hard to add on to.

Reply:
I agree that it is a bit more complex, but it is also finally future proof. We were before relying solely on an array index for the toggles correlating to the cell index we wanted. Because of the new locations of cells that can change (ie. be removed or added from settings), that it now no longer an option. This will allow us to support having settings that vary significantly more also in the future (for example, when we add Siri Shortcuts for iOS 12+ users, we will be able to display that without having to worry about changing subsequent indices in the array).

Reply:
Makes sense, thanks for explaining!